### PR TITLE
doc: How to develop

### LINKED ISSUE #61: Improving the onboarding of new contributors
NuGet Trends is small but it has a few moving parts, which makes getting started contributing with the project not that straightforward.

The main "issue" is data. Without some meaningful amount of data, it's hard to develop anything on the site.

This issue is to track ideas and how we can improve the onboarding of new contributors. A few ideas we have (mainly from @bruno-garcia)

- Create a docker image which contains some loaded data into a Postgres DB
- Create a `CONTRIBUTING.md` file, stating how people can get started helping us
- Improve the current `README.md` with some info about the project, or whatever else. 

What else?

### PR DESCRIPTION
**Pull request by bruno-garcia**
Resolves #61 
We still want to publish on `hub.docker.com` actual `postgres` images with data in it but that'll be a separate workitem.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,39 @@
...
+1. Run `docker-compose up` to get the services.
+
+And either:
+
+2. Restore a DB backup to get some data, like the packages and some download number.
+
+Or:
+
+2. Run _Entity Framework_ migrations to get the database schema in place.
+3. Run the _catalog importer_ from scratch.
```
Comment (loic-sharma) [inline README.md:32]: How do you run these importers? Is this automatically done by starting the `NuGetTrends.Scheduler` app?

**Comment by loic-sharma** [comment]
Thanks for adding this guide!

File: README.md (hunk 1 above)
Comment (bruno-garcia) [reply README.md:32]: Yes. It's scheduled with Hangfire.
Good point! It's not clear.

Comment (bruno-garcia) [reply README.md:32]: Added more details

Comment (bruno-garcia) [reply README.md:32]: I've described you can go to the dashboard and run them manually. Let me know if it's clear now

Comment (loic-sharma) [reply README.md:32]: Perfect, thanks!

**Pull request merged**